addons/mail/static/tests/tours/mail_chatter_activity_test_tour.js

```diff
@@ -0,0 +1,36 @@
+/* @odoo-module */
+
+import { registry } from "@web/core/registry";
+
+
+registry.category("web_tour.tours").add("mail_activity_schedule_from_chatter", {
+    test: true,
+    steps: [
+        {
+            trigger: "button:contains('Activities')",
+        },
+        {
+            trigger: "input[id='summary_0']",
+            run: "text Play Mario Kart",
+        },
+        {
+            trigger: "button:contains('Mark as Done')",
+        },
+        {
+            trigger: '.o-mail-Message-body:contains("Play Mario Kart")',
+        },
+        {
+            trigger: "button:contains('Activities')",
+        },
+        {
+            trigger: "input[id='summary_0']",
```

Comment:
Could you find a better selector than `summary_0`?
I don't get where the `0` comes from. It leaks a technical detail that you make test no longer pass when functionally it still works.

If you find a better selector, could prevent frustration in wasting an afternoon or a day because this has been changed for some reasons!